test/e2e/support/timeout.js

```diff
@@ -0,0 +1,6 @@
+// timeout.js
+var { setDefaultTimeout } = require("cucumber");
+
+setDefaultTimeout(600 * 1000);
```

Comment:
setting to 10 mins here and 1 min in world.js is this correct?

Reply:
these values are not used world.js and timeout.js, as we used the same old files which we will be refactoring in our tidyup tasks